command/agent/cache/cacheboltdb/bolt.go

```diff
@@ -242,7 +245,12 @@ func (b *BoltStorage) GetAutoAuthToken(ctx context.Context) ([]byte, error) {
 		return nil, nil
 	}
 
-	plainText, err := b.keyMgr.Decrypt(ctx, token, []byte(b.aad))
+	var blob wrapping.EncryptedBlobInfo
+	if err := proto.Unmarshal(token, &blob); err != nil {
+		return nil, err
+	}
+
+	plainText, err := b.wrapper.Decrypt(ctx, &blob, []byte(b.aad))
```

Comment:
Nit: we should rename this to `plaintext` (same with `ciphertext` where applicable). We can do this on the main branch after merging of this PR to avoid merge conflicts.